.gitignore

```diff
@@ -21,6 +21,7 @@ coverage
 # Release output
 dist/**
 operator-controller.yaml
+olmv1.yaml
```

Comment:
WDYT about leaving the manifest file called `operator-controller.yaml` for now? And then we tackle the rename in a follow-up?